File: MyApplication/app/src/main/java/com/example/ankomah/myapplication/MainActivity.java
package com.example.ankomah.myapplication;

import android.os.AsyncTask;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;

import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.net.HttpURLConnection;

import javax.net.ssl.HttpsURLConnection;

public class MainActivity extends AppCompatActivity {

    EditText num1,num2;
    Button btn;

String strUrl =
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        num1 =(EditText) findViewById(R.id.num1);
        num2 =(EditText) findViewById(R.id.num2);

        btn = (Button) findViewById(R.id.btnAdd);
        btn =setOnClickListner(new View.OnClickListener(){
            @Override
            public void onClick(View v)
            {
                int i = Integer.parseInt(num1.getText().toString());
                int j = Integer.parseInt(num2.getText().toString());

                new MultiplyTask().execute();
            }
        });
    }
    public class MultiplyTask extends AsyncTask<String,String,String>
    {
        @Override
        protected void onPreExecute() {
            super.onPreExecute();
        }

        @Override
        protected void onPostExecute(String s) {
            super.onPostExecute(s);
        }

        @Override
        protected String doInBackground(String... strings) {

            try{
                URL url =new URL(strUrl);
                HttpURLConnection con =(HttpsURLConnection) url.openConnection();
                con.setRequestMethod("GET");
                con.connect();

                BufferedReader bf =new BufferedReader(new InputStreamReader()));
                String Value = bf.readline();
                System.out.println("result is" + value);
            }
            catch (Exception e)
            {
                System.out.println(e);
            }
            return null;
        }
    }
}

File: UpdatingUI/app/src/main/java/com/example/ankomah/updatingui/MainActivity.kt
package com.example.ankomah.updatingui

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.os.Handler
import android.widget.TextView
import java.util.*

public class MainActivity extends AppCompatActivity() {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
    TextView myview;
    int x;
    Random myRand = new Random();
    String number;
    Handler myhandler = new Handler;

    public void randomize(view view) {
        Runnable myRunnable = new Runnable() {
            @Override
            public void run() {
                generateNumber();
            }

        };
        new Thread (myRunnable).start();

    }
    private void generateNumber() {
        for (int i = 0; i <= 40; i++){
        x = myRand.nextInt(100) + 1;
        number = String.valueOf(x);
        try {
            Thread.sleep(50);
        } catch (InterruptedException e) {
            e.printStackTrace();
        }
        myviewHandler.post(new Runnable) {
            @Override
            public void run() {
                myView = (TextView) findViewById (R.id.myTextView)
                myView.setText(number);


            }
        });


    }
    }
}


